test/src/runner/cucumber-integration/testCliArgs.js

```diff
@@ -0,0 +1,75 @@
+const assert = require('assert');
+const path = require('path');
+const mockery = require('mockery');
+const common = require('../../../common.js');
+const CucumberRunner =  common.require('runner/test-runners/cucumber.js');
+
+
+
+
```

Comment:
too many blank lines, can you run eslint. It will fix small issues like this

Reply:
Will remove it. ESlint has no rule for blank lines